### DISCUSSION TITLE
Daily In/Out

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#21] [Q&A]
How many pets come into animal quarantine each day?

How many are picked up? 

How many have to stay over night?

**Comment by JJ-Foust** [comment]
60-75 per day

**Comment by judyn808** [comment]
Average 100 pets enter each day between 8AM and 5PM. Not sure how many come in after hours.

**Reply by JJ-Foust** [reply]
@judyn808 where did you get 100. I asked chris on slack and got the answer of 60-75 between 8:30 and 4:30. Did the say something else on saturday?

**Reply by judyn808** [reply]
I got it from the video that was posted in the channel.

**Reply by judyn808** [reply]
100 animals , 8:30-5PM. Maybe 100 is on a hairy day.

**Reply by JJ-Foust** [reply]
Or maybe they don't really know and are just guessing. LOL

**Reply by JJ-Foust** [reply]
I was asking for the purposes to see approximately how many texts/emails would be sent daily. To estimate cost for a text service.

**Reply by JJ-Foust** [reply]
Just fail conservative and use the 100 per day
